src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/InnerAssignmentCheck.java

```diff
@@ -57,6 +57,34 @@
  * <pre>
  * &lt;module name=&quot;InnerAssignment"/&gt;
  * </pre>
+ * <pre>
```

Comment:
`<p>Example:</p>` is missing

Reply:
Thanks for the feedback. I've added this